Gopkg.toml

```diff
@@ -84,6 +84,13 @@
   name = "github.com/docker/docker"
   revision = "a9fbbdc8dd8794b20af358382ab780559bca589d"
 
+# This override enables to use quantity.Resource as type during generation of OpenAPI specs
+# (https://github.com/kubernetes-sigs/controller-tools/pull/174)
+# TODO: avoid revision once a release containing this feature has been rolled out
```

Comment:
Can you open an issue for this, if/once this is merged so we keep track and don't forget to do this?

Reply:
@LiliC we're lucky enough :)
[v0.1.10](https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.1.10) has been released! :tada: 
Updating PR